settings.gradle.kts

```diff
@@ -0,0 +1,24 @@
+rootProject.name = "kotlin-full-stack-application-demo"
+
+pluginManagement {
+    resolutionStrategy {
+        eachPlugin {
+            if (requested.id.id == "kotlinx-serialization") {
```

Comment:
It can be omitted for `org.jetbrains.kotlin.plugin.serialization` because it is in Gradle plugin portal (https://plugins.gradle.org/plugin/org.jetbrains.kotlin.plugin.serialization)